app/components/channel_drawer_list/channel_drawer_list.js

```diff
@@ -125,7 +126,15 @@ class ChannelDrawerList extends Component {
 
     handleClose = (channel) => {
         this.setState({showOptions: false});
-        this.props.actions.closeDMChannel(channel);
+
+        switch (channel.type) {
+        case Constants.DM_CHANNEL:
+            this.props.actions.closeDMChannel(channel);
```

Comment:
Should we maybe make a single action for closing a DM/GM channel that handles figuring out which type it is? I noticed you're also doing this below